File: benchmark/fib.py

def fib(n):
    if n == 0:
        return 0;
    if n == 1:
        return 1;
    l = fib(n - 1)
    r = fib(n - 2)
    return l + r

if __name__ == "__main__":
    print(fib(40))

File: benchmark/fib.c
#include <stdio.h>

long long fib(int n){
    if(n == 0) return n;
    if(n == 1) return n;
    long long l = fib(n-1);
    long long r = fib(n-2);
    return l + r;
}

int main(int argc, char* argv[]){
    printf("%d\n", fib(atoi(argv[1])));
    return 0;
}
